miden-crypto/src/merkle/mmr/partial.rs

```diff
@@ -84,6 +84,16 @@ impl PartialMmr {
         Self { forest, peaks, nodes, track_latest }
     }
 
+    /// Returns a new [`PartialMmr`] instantiated with empty peaks and nodes and 0 leaves.
+    pub const fn empty() -> Self {
```

Comment:
Do we need this to be `const`? If not, then I'd probably prefer implementing `Default` trait from `PartialMmr`.

Reply:
The const is not necessary. Wasn't sure if `Default` impl was appropriate here - will do now